structs.go

```diff
@@ -780,6 +780,11 @@ type Member struct {
 	// The nickname of the member, if they have one.
 	Nick string `json:"nick"`
 
+	// The member's guild avatar hash.
+	// This avatar is only shown inside the guild this member is a part of.
+	// NOTE: This is not documented in the API yet. Guild-specific avatars are subject to change.
+	Avatar string `json:"avatar"`
```

Comment:
`Avatar` is marked as nullable in the docs; it should probably be `*string` to avoid issues in unmarshaling.